lib/hash.js

```diff
@@ -83,9 +83,13 @@ LibHash.prototype.computeHash = function(path) {
                         if (er1 || er2) {
                             return reject(er1 || er2);
                         }
-                        checksumReady(self.checksumBuffer(buf1, 16));
-                        checksumReady(self.checksumBuffer(buf2, 16));
-                        fs.close(fd);
+                        fs.close(fd, function(err) {
+                            if (err) {
+                                return reject(err);

```

Comment:
By the time fs.close is called, every needed portion of the file is already read into the buffer. Nothing will break I suppose.